src/Microsoft.Identity.Web/TokenCacheProviders/Distributed/MsalDistributedTokenCacheAdapter.cs

```diff
@@ -65,6 +68,18 @@ public partial class MsalDistributedTokenCacheAdapter : MsalAbstractTokenCachePr
             }
         }
 
+        private static IServiceProvider? GetServiceProvider(
```

Comment:
Maybe add a little comment to say that the method returns the service provider if the MsalDistributedTokenCacheAdapterOptions.Encrypt property is true, and otherwise null?

Reply:
sure.